### PR TITLE
Created bigInsights recipe

### PR DESCRIPTION
**Pull request by yoramw**

### CODE REVIEW

**Comment by uric** [comment]
Please change the recipes to be included in the services folder. The apps folder is just a way to compose multiple service recipes and should not contain complete recipes.

**Comment by uric** [comment]
Also, please do the following: 
- Your README should have a .md extension so that its markup is processed properly 
- Add a few more words about what this application recipe is about and what it does 
- Add a README.md file to each service, that describes what it is and where it was tested (same is we did for the other services). Also describe what main properties are there and what you expect the user to config
- Add apache2 license header to all files except the readme

File: apps/biginsights/data/data_start.groovy [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import org.cloudifysource.dsl.context.ServiceContextFactory
+import java.util.concurrent.TimeUnit
+
+println "data_start.groovy: BigInsights data node is about to start"
+
+sleep 7200000
```
Comment (uric) [inline apps/biginsights/data/data_start.groovy:6]: What is this sleep used for?

File: apps/biginsights/flume/flume_start.groovy [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import org.cloudifysource.dsl.context.ServiceContextFactory
+import java.util.concurrent.TimeUnit
+
+println "flume_start.groovy: BigInsights flume node is about to start"
+
+sleep 7200000
```
Comment (uric) [inline apps/biginsights/flume/flume_start.groovy:6]: Why do we need this?

File: apps/biginsights/flume/hp-cloud-demo.pem [hunk 3]
```diff
@@ -0,0 +1,15 @@
```
Comment (uric) [inline apps/biginsights/flume/hp-cloud-demo.pem:0]: You shouldn't inclue a private key file in the repo!

File: apps/biginsights/biginsights-application.groovy [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+		name = "data"
+	}
+/*	service {
+		name = "hbase"
+	}
+	service {
+		name = "flume"
+	}
+	
+	service {
```
Comment (uric) [inline apps/biginsights/biginsights-application.groovy:19]: Please remove it altogether

**Comment by uric** [comment]
Seems all the services except master are dummies, is that correct?

File: apps/biginsights/data/data_start.groovy (hunk 1 above)
Comment (yoramw) [inline apps/biginsights/data/data_start.groovy:6]: It is leftovers. I will remove it

**Comment by yoramw** [comment]
They are not dummies, but rather preparations for the master to deploy the relevant services there.
It does do important preparation work.

Flume and hive are currently not in use (were used for a demo Nati wanted to show the services spreading over many machines. If we could colocate services on the same server, I would have left them running, but otherwise it is a waste.

Yoram

**Pull request closed without merging**